flink-table/flink-table-planner/pom.xml

```diff
@@ -196,14 +189,15 @@ under the License.
 			<!-- For legacy string expressions in Table API -->
 			<groupId>org.scala-lang.modules</groupId>
 			<artifactId>scala-parser-combinators_${scala.binary.version}</artifactId>
+			<scope>provided</scope>
```

Comment:
Not sure if this is provided. It is not contained in Scala 2.12 by default.

Reply:
It doesn't matter, as we package it in table planner, so it doesn't need to be pulled in transitively for modules depending on `flink-table-planner` directly.